### DISCUSSION TITLE
How to send a Kibana field value in the email tag in elastalert

### DISCUSSION THREAD

**Discussion by Singh-Shivani** [jertel/elastalert2#777] [Q&A] [answered]
Hi, 
I want to send email alert to a list of people which is coming from kibana in the field `mail_to `and `mail_cc` : 

I have two fields -

1. `mail_cc: ` contains multiple email id separated by comma
2. `mail_to:  `contains single email id 

![image](https://user-images.githubusercontent.com/47295558/159712727-b50f3d09-6eca-44d5-b035-d63b629e4774.png)
I want to show my configuration like this - 
> alert:
> - "email"
> email: mail_to
> cc :  mail_cc

Although this is wrong sytanx, how can I get my results?

**Comment by jertel** [comment, marked as answer]
From the docs at https://elastalert2.readthedocs.io/en/latest/ruletypes.html#email

![image](https://user-images.githubusercontent.com/16639611/159720257-d0762819-a7ac-489a-92a9-47a8433362c2.png)

I don't know if there's a similar, undocumented field for cc, but if you you can consider submitting a pull request, if interested.

**Reply by Singh-Shivani** [reply]
As per to my understanding, `email_from_field` can only hold 1 email id.
My case includes list of email id.  I'm sending emails like this - 
![image](https://user-images.githubusercontent.com/47295558/159968952-147f7498-b9aa-4dd4-bb5e-094b9d1c472c.png)

**I wish to include this email id list from one the field being passed to elastalert from Kibana index**

**Reply by jertel** [reply]
It's confusing because your first message stated that the email_to field only contains one email address:
![image](https://user-images.githubusercontent.com/16639611/159973615-1ebe4624-fb4f-47b2-8b69-9238ed90ad8e.png)

Regardless, per the documentation I pasted above, this `email_from_field` can be either a single email address, or a list of email addresses.

**Reply by Singh-Shivani** [reply]
**Can you please give the right syntax in which I can send email-ids in rule file?**
If field `mail_id` is coming from kibana contains data `"test@gmail.com", "test2@gmail.com"`

Earlier I wrote this but it didn't work, rest of my config was same: 
`email: mail_id`

In alert mails I got like this -
![image](https://user-images.githubusercontent.com/47295558/159975942-93193dfd-2e0d-4e62-a703-8cb1d1266671.png)

**Reply by jertel** [reply]
Your rule config should have this:

```
email_from_field: "mail_to"
```

Then your elasticsearch document should store the email address as a list/array type inside of the `mail_to` field.

**Reply by Singh-Shivani** [reply]
okay
